cdap-watchdog/src/main/java/io/cdap/cdap/logging/clean/LogCleaner.java

```diff
@@ -27,25 +33,37 @@
 /**
  * cleanup expired log files
  */
-public class LogCleaner implements Runnable {
+public class LogCleaner {
   private static final Logger LOG = LoggerFactory.getLogger(LogCleaner.class);
+  private static final String FOLDER_DATE_FORMAT = "yyyy-MM-dd";
```

Comment:
Where is it defined when we generate it? It's better not to duplicate it

Reply:
It seems we have it in io.cdap.cdap.logging.appender.system.LogFileManager#getLocation. Can we reuse a constant for `yyyy-MM-dd`

Reply:
This would help to ensure it's consistent

Reply:
Done